test/rekt/resources/broker/broker.go

```diff
@@ -159,7 +159,7 @@ func IsAddressable(name string, timings ...time.Duration) feature.StepFn {
 }
 
 // Address returns a broker's address.
-func Address(ctx context.Context, name string, timings ...time.Duration) (*apis.URL, error) {
```

Comment:
I'm just trying to get a better idea of what's going on in the rekt tests, why are we switching from `apis.URL` to `duckv1.Addressable`? Is this because of an api change in the `reconciler-test` package?

Reply:
Yes, this particular change is mostly aligning to the rekt API, in rekt, we needed CA certs from Addressable and having only URL wasn't enough.

Reply:
That makes sense, thanks for the clarification!